metron-deployment/roles/monit/templates/monit/monit.conf

```diff
@@ -0,0 +1,29 @@
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+set logfile syslog
+set pidfile /var/run/monit.pid
+
+# check services at X second intervals
+set daemon 30
+include /etc/monit.d/*
+
+# allow http access
+set httpd port 2812 and
+    use address {{ ansible_host }}
```

Comment:
Good deal, thanks for checking!

On Wed, May 11, 2016 at 10:40 PM, Nick Allen <notifications@github.com>
wrote:

> In metron-deployment/roles/monit/templates/monit/monit.conf
> <https://github.com/apache/incubator-metron/pull/118#discussion_r62957806>
> :
>
> > +#  distributed under the License is distributed on an "AS IS" BASIS,
> > +#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +#  See the License for the specific language governing permissions and
> > +#  limitations under the License.
> > +#
> > +
> > +set logfile syslog
> > +set pidfile /var/run/monit.pid
> > +
> > +# check services at X second intervals
> > +set daemon 30
> > +include /etc/monit.d/*
> > +
> > +# allow http access
> > +set httpd port 2812 and
> > +    use address {{ ansible_host }}
>
> Double-checked. The external host name is added to /etc/monit.conf as
> expected.
>
> I believe this is due to how we have the Ansible/EC2 module configured; in
> the ec2.ini file. In there we tell it to use the external host name instead
> of the internal.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-metron/pull/118/files/f02229de7acee960ba4d932da0cfd8e0e8049cc2#r62957806>
>